app/javascript/elm/src/Main.elm

```diff
@@ -331,14 +339,42 @@ update msg model =
         ShowCopyLinkTooltip tooltipId ->
             ( model, Ports.showCopyLinkTooltip tooltipId )
 
-        ShowPopup items itemType selectedItem ->
-            ( { model | popup = Popup.SelectFrom items itemType selectedItem, isPopupExtended = False, overlay = Overlay.update (AddOverlay PopupOverlay) model.overlay }, Cmd.none )
+        ShowListPopup popup ->
+            ( { model | popup = popup, isPopupListExpanded = False, overlay = Overlay.update (AddOverlay PopupOverlay) model.overlay }, Cmd.none )
+
+        ShowExportPopup ->
+            ( { model | popup = Popup.EmailForm }, Cmd.none )
```

Comment:
There is this one small thing that has to do with the overlays - 
on session list view click on paramter - you have a list popup and a gray overlay over the map
- then click on export sessions - the list popup dissapears but the gray overlay remains.
Not a big thing, so not sure if it's worth changing, unless that's something easy. I think adding `overlay = Overlay.update (RemoveOverlay PopupOverlay) model.overlay` could work. (similar to what we have in `ClosePopup` branch)